### ISSUE TITLE
Register holding down a button of the mouse

### ISSUE THREAD

**Issue by RMPR** [RMPR/atbswp#91]
## Checklist

- [x] I use the latest release of atbswp
- [x] The issue is not existing yet

### Operating System
Any

### Desktop Environment/Window Manager
Any

### Python version
Any

## Description

When holding down a button of the mouse, the action is still registered as one click.

**Comment by qkum** [comment]
maybe this is the first time you wrk with controlling the mouse

from my experience this is how it should work

here is what the pc sees from my understanding:
```
pyautogui.mouseDown(409, 604, 'left')        - _you Start pressing 'left' mouse button down_, at coordinates: X=409, Y=604
time.sleep(10.0)                                              - _you press the button down for 10 sek_, aka sleep for 10 sek
pyautogui.moveTo(454, 608)                          - _you move the mouse_ to coordinates: X=454, Y=608 
pyautogui.mouseUp(454, 608, 'left')             - _you Stop pressing 'left' mouse button down_, at coordinates: X=454, Y=608
```
when you as a human press down a button for 10 sek and then stop. are you then clicking 1 time down and 1 time up, or 10 times down and up? Nope. 

`1 time down, pause for 10 sek, 1 time up` is what you are doing in a computers world

if what you really are talking about is the program clicking the mouse button Down and Up instead of holding the button down. then i can just tell you that it works fine on my windows 10 pc

i just did a test with paint. pressing left mouse button down while drawing a circle and it works perfect. it does not click instead of holding down the mouse button for me.

**- in short, program seems to work as it should and the bug report must come dwn to lack of knoweldge or dependencies/OS problems**

**Comment by qkum** [comment]
**- in future, please add an example of the problem/error!**

**Comment by RMPR** [comment]
Thanks for helping triaging the issue, I tried to reproduce it just now, and it seems like it might have been a bug with Windows. I think you should tailor your way of talking as to not appear aggressive to a common person creating an issue. 
For example:

> maybe this is the first time you wrk with controlling the mouse

This wasn't needed, it was needlessly assuming knowledge (or lack thereof) from the user. People tend to react badly to this kind of comment.

>  and the bug report must come dwn to lack of knoweldge or dependencies/OS problems 

The same with this part here, if you stopped at "works as it should". The meaning is conveyed, the rest might be taken in a weird way, and imho, it doesn't add anything of value to the point you're trying to make.


Again, thanks for helping triaging the issue especially since I didn't have time to really maintain the project these days.
